src/sentry/utils/members.py

```diff
@@ -0,0 +1,92 @@
+from django.conf import settings
```

Comment:
Nit: It might be a good idea to add tests for this file, now that this functionality is stand-alone.

Reply:
@wedamija good call, I can add some basic tests